central/declarativeconfig/health/datastore/datastore_impl.go

```diff
@@ -32,10 +33,17 @@ func (ds *datastoreImpl) GetDeclarativeConfigs(ctx context.Context) ([]*storage.
 }
 
 func (ds *datastoreImpl) UpsertDeclarativeConfig(ctx context.Context, configHealth *storage.DeclarativeConfigHealth) error {
+	if err := ds.verifyContextKey(ctx); err != nil {
+		return err
+	}
 	return ds.store.Upsert(ctx, configHealth)
 }
 
 func (ds *datastoreImpl) RemoveDeclarativeConfig(ctx context.Context, id string) error {
+	if err := ds.verifyContextKey(ctx); err != nil {
```

Comment:
`verifyContextKey` is a very abstract name. How about `verifyDeclarativeContext`?